app/helpers/template_helper.rb

```diff
@@ -0,0 +1,14 @@
+  module TemplateHelper
+    def build_remediation_plan(remediation_ids, host)
+      entries = PreupgradeReportEntry.where(id: remediation_ids.split(','), host_id: host.id)
```

Comment:
Wouldn't it be better if the macro expected an array of ids instead of a string wich comma separated ids?

Reply:
but this is user input, that won't be an array right? would you prefer to always split?

`build_remediation_plan(input('ids').split(','))`

Reply:
It cannot be an array because it's value from user input field, which will always be string.

Reply:
What I was suggesting was doing the splitting in the template, instead of in the macro itself

Reply:
Ok, I'll move the logic to template.